data/Makefile

```diff
@@ -9,11 +9,13 @@ define get_repmasker
 	gzip -c -d $(1)/$(1).fa.out.gz | poetry run parse_rm - > $(1).bed
 endef
 
-all: hg19 hg38 mm10/chr2.fa.gz.npz mm10.bed deepgrp_train mm10.chrom.sizes
+all: hg19 hg38 mm10/chr2.fa.gz.npz mm10.bed deepgrp_train mm10.chrom.sizes hg19.bed
 
 mm10.bed:
 	$(call get_repmasker,mm10)
 
+hg19.bed:
```

Comment:
The in the paper we used a custom `hg19.bed´ as explained here: https://github.com/fhausmann/deepgrp_reproducibility/blob/04dac5bee6577b35c63a595a58330d2ca40d7aa5/repeatmasker/README.txt

In general a hg19.bed like with your command should work as well, but could give slightly different results, therefore I think it would be best to not add this command in the Makefile.

Reply:
I removed the rule again